app/controllers/profile/friends_controller.rb

```diff
@@ -0,0 +1,15 @@
+class Profile::FriendsController < ApplicationController
+  def index
+    @friends = current_user.load_friends
+  end
+
+  def create
+    if current_user.add_friend(params[:email_address])
+      current_user.reload
+      flash[:success] = 'Friend Added Successfully'
```

Comment:
Wonder if this could be more verbose - if we could get username from the email. `"John Deer was added as a friend"` or something